python-package/xgboost/core.py

```diff
@@ -1721,7 +1714,7 @@ def get_split_value_histogram(self, feature, fmap='', bins=None, as_pandas=True)
         xgdump = self.get_dump(fmap=fmap)
         values = []
         regexp = re.compile(r"\[{0}<([\d.Ee+-]+)\]".format(feature))
-        for i in range(len(xgdump)):
+        for i, _ in enumerate(xgdump):
```

Comment:
Just out of curiosity, why using `enumerate`.

Reply:
To comply with pylint style guideline: PyCQA/pylint#684